Tools/YamlCreate.ps1

```diff
@@ -310,7 +310,7 @@ Function Request-Installer-Url {
 # Prompts the user to enter installer values
 # Sets the $script:Installers value as an output
 # Returns void
-Function Read-WinGet-InstallerValues {
+Function Read-Installer-Values {
```

Comment:
Isn't two dashes in a PowerShell function name against the law?

Maybe this should be changed to `Read-InstallerValues`. 

(Sorry I didn't see your initial PR.)

Reply:
I don't know why it would be against the law? The [current version](https://github.com/microsoft/winget-pkgs/blob/a51d74b195208f815909a86f2aed5dd75fbc46a6/Tools/YamlCreate.ps1#L84) of the script has 2 dashes in function names. It seems to work just fine. Its not difficult to change though if we want to

Reply:
I can't find a citation, but I think Microsoft recommends using a single dash as above to stick to the `Verb-Noun` pattern they started. 

Since these functions aren't for external use outside the script anyway, I guess it doesn't matter. just something I randomly noticed.